setupext.py

```diff
@@ -627,6 +627,13 @@ def do_custom_build(self, env):
                 },
                 **env,
             }
+            if os.path.exists(os.path.join(src_path, "autogen.sh")):
+                try:
+                    subprocess.check_call(["sh", "./autogen.sh"], env=env, cwd=src_path)
```

Comment:
In almost all cases this will be a spurious warning.  We have not been using autotools from at least 2015.

Can you please point to documentation from freetype that when installing from the release tarball it is expected that running `autogen.sh` is required?

Reply:
Alright, i find that autogen.sh is used to generate `configure.ac`.So just run it if this file does not exist.